src/Speckle.Objects/BuiltElements/Rebar.cs

```diff
@@ -1,16 +1,15 @@
-using System;
-using System.Collections.Generic;
-using Objects.Geometry;
-using Speckle.Core.Models;
+using Speckle.Objects.Geometry;
+using Speckle.Sdk.Models;
 
-namespace Objects.BuiltElements;
+namespace Speckle.Objects.BuiltElements;
 
 /// <summary>
 /// A reinforcement bar group comprised of reinforcing bars of the same type and shape.
 /// </summary>
 /// <remarks>
 /// This class is not suitable for freeform rebar, which can have multiple shapes.
 /// </remarks>
+[SpeckleType("Objects.BuiltElements.RebarGroup")]
 public class RebarGroup<T> : Base, IHasVolume, IDisplayValue<List<ICurve>>
```

Comment:
we need not fix it for this pr, but you've reminded me, we ought to enforce that any generic typed base objects need to be abstract, or the speckle_type starts to include assembly versioning information in a nasty way...

Reply:
this is will be abstract